packages/build-tools/src/customBuildContext.ts

```diff
@@ -56,6 +59,9 @@ export class CustomBuildContext implements ExternalBuildContextProvider {
     this.runtimeApi = {
       uploadArtifacts: (...args) => buildCtx['uploadArtifacts'](...args),
     };
+    this.cacheManager = buildCtx.cacheManager;
```

Comment:
We could consider rolling `cacheManager` into `runtimeApi` for simplicity. Like, how many functions may we put into `runtimeApi`? I don't think the list will be too long.